shell/components/formatter/PodsUsage.vue

```diff
@@ -8,47 +7,17 @@ export default {
       required: true
     },
   },
-  data() {
-    return {
-      loading:   true,
-      podsUsage: null
-    };
-  },
-  methods: {
-    async startDelayedLoading() {
-      const id = this.row?.mgmt?.id;
-
-      if (this.row?.isReady && id) {
```

Comment:
this is lost, so we now show `0 / 0` for clusters that aren't up

Reply:
Now checks for "isReady" and shows spinner if not